manifests/nodejs.yml

```diff
@@ -804,7 +804,10 @@ tests/:
     test_debugger_exception_replay.py:
       Test_Debugger_Exception_Replay: missing_feature (feature not implented)
     test_debugger_expression_language.py:
-      Test_Debugger_Expression_Language: missing_feature (feature not implented)
+      Test_Debugger_Expression_Language:
+        '*': irrelevant
```

Comment:
not sure why you put irrelevant for other weblogs ? usually this should be missing_feature instead. Irrelevant means it really will make no sense to ever run this test for perpetuity.

Reply:
Using `irrelevant` seems to be the predominant pattern to skip `*` (used 25 times in this file compared to 5 times for `missing_feature`).

`irrelevant` means that it will not try to run the tests (faster CI runs) whereas `missing_feature` means that it will run the tests and report it if it suddenly starts passing. I need to create `/debugger/*` http endpoints in the other apps for them to be testable. Since I haven't done that yet, they will just fail and there's no chance of them suddenly working by themselves. Once I do, I know to come here and enable them.

Reply:
alright, fair point